src/Record.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class Record {
+
+    private string $album_name;
+    private string $artist_name;
+    private int $year;
+    private string $record_label;
+    private int $number_of_tracks;
```

Comment:
like methods, properties should be camelCase

Reply:
in general camelCase is the way for most things, especially in OOP

Reply:
changed in object and test